src/Microsoft.Diagnostics.Monitoring.HostingStartup/ParameterCapturing/BoxingInstructions.cs

```diff
@@ -58,20 +30,20 @@ public static bool[] AreParametersSupported(uint[] tokens)
             return supported;
         }
 
-        public static uint[] GetBoxingTokens(MethodInfo method)
+        public static ParameterBoxingInstructions[] GetBoxingInstructions(MethodInfo method)
         {
             List<Type> methodParameterTypes = method.GetParameters().Select(p => p.ParameterType).ToList();
-            List<uint> boxingTokens = new List<uint>(methodParameterTypes.Count + (method.HasImplicitThis() ? 1 : 0));
-
+            List<ParameterBoxingInstructions> instructions = new(methodParameterTypes.Count + (method.HasImplicitThis() ? 1 : 0));
             //
             // A signature decoder will used to determine boxing tokens for parameter types that cannot be determined from standard
             // reflection alone. The boxing tokens generated from this decoder should only be used to fill in these gaps
             // as it is not a comprehensive decoder and will produce UnsupportedParameterToken for any types not explicitly mentioned
```

Comment:
nit: It looks like `UnsupportedParameterToken` is no longer being defined/referenced in other places, so this should probably be updated